spec/order_list_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'order_list'
+
+describe OrderList do
+
+  let(:pizza){double(:pizza, name: :pizza, price: 1)}
+  let(:chicken){double(:chicken, name: :chicken, price: 2)}
+  let(:dish_list){double(:dish_list, prices: {pizza: 1, chicken: 2}, dishes: [pizza, chicken])}
```

Comment:
Metrics/LineLength: Line is too long. [95/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)